src/scheduler.cc

```diff
@@ -223,7 +224,7 @@ Status SchedulerService::TaskInfo(ServerContext* context, const TaskInfoRequest*
   }
   for (int i = 0; i < workers->size(); ++i) {
     OperationId operationid = (*workers)[i].current_task;
-    if (operationid != NO_OPERATION) {
+    if (operationid != NO_OPERATION && operationid != ROOT_OPERATION) {
```

Comment:
Let's keep it in the back of our minds and if we continue to need it then we should make it a function.